app/models/competition.rb

```diff
@@ -481,6 +502,7 @@ def confirmed_or_visible?
     self.confirmed? || self.showAtAll
   end
 
+  # TODO: Consider refactoring this once auto-accept is implemented
```

Comment:
Refactor to what purpose?

Reply:
I genuinely have no idea - comment removed, and lesson learned about including the details of my thoughts when I make comments like this